src/NServiceBus.Metrics.ServiceControl.Tests/NServiceBus.Metrics.ServiceControl.approved.cs

```diff
@@ -0,0 +1,15 @@
+﻿[assembly: System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.5.2", FrameworkDisplayName=".NET Framework 4.5.2")]
+
+namespace NServiceBus.Metrics.ServiceControl
+{
+    
+    public class static ReportingConfigurationExtensions
+    {
+        public static NServiceBus.Metrics.ServiceControl.ReportingOptions EnableReporting(this NServiceBus.Settings.SettingsHolder settings) { }
```

Comment:
Do we need `EnableReporting` methods? I think that they are just noise until we have more than one `SendMetricDataToServiceControl` method.